src/components/CalendarTagTwoStep/CalendarTagTwoStep.js

```diff
@@ -163,12 +163,12 @@ CalendarButtonTwoStep.propTypes = {
   confirmText: PropTypes.string
 };
 
-CalendarButtonTwoStep.defaultProps = {
+CalendarTagTwoStep.defaultProps = {
   clearText: 'Clear',
   confirmText: 'Apply'
 };
 
 /**
  * @component
  */
-export default CalendarButtonTwoStep;
+export default CalendarTagTwoStep;
```

Comment:
For a class component, I feel it's totally fine to do `export default class CalendarTagTwoStep`.